Code/Framework/AzCore/AzCore/Math/ShapeIntersection.inl

```diff
@@ -120,6 +120,12 @@ namespace AZ
         }
 
 
+        AZ_MATH_INLINE bool Overlaps(const Sphere& sphere, const Capsule& capsule)
+        {
+            return Overlaps(capsule, sphere);
+        }
+
+
```

Comment:
nit: extra carriage return here.

Reply:
I don't really like it, but that is the existing style in that file.